contracts/examples/DXswapRelayer.sol

```diff
@@ -310,27 +307,21 @@ contract DXswapRelayer {
         uint256 _minA,
         uint256 _minB
     ) internal {
-        if (_tokenA != address(0) && _tokenB != address(0)) {
-            TransferHelper.safeApprove(_pair, dxSwapRouter, _liquidity);
-            IDXswapRouter(dxSwapRouter).removeLiquidity(
-                _tokenA,
-                _tokenB,
-                _liquidity,
-                _minA,
-                _minB,
-                address(this),
-                block.timestamp
-            );
-        } else {
-            TransferHelper.safeApprove(_pair, dxSwapRouter, _liquidity);
-            IDXswapRouter(dxSwapRouter).removeLiquidityETH(
-                _tokenB,
-                _liquidity,
-                _minA,
-                _minB,
-                address(this),
-                block.timestamp
-            );
+        _tokenA = _tokenA == address(0) ? WETH : _tokenA;
```

Comment:
is this really needed? if we will just use WETH, and it will be the tokenA address, thats fine.